.github/workflows/github_autodeploy_action.yml

```diff
@@ -74,7 +74,7 @@ jobs:
      with:
       context: .      
       push: true
-      tags: swapifyapp/swapify-app
+      tags: swapifyapp/swapify-app:production
```

Comment:
I suggest, this will tag the version in master repo, do we have also som version info, which could be assigned - version on git will match version of the application.